packages/webpack-plugin/src/runtime.ts

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable no-var */
```

Comment:
with project references, we'll be able to set es5 specifically to the runtime.

Reply:
cool

Reply:
for now keeping